synopsys-task/src/synopsys-task/synopsys-bridge.ts

```diff
@@ -73,6 +73,35 @@ export class SynopsysBridge {
     }
 
     try {
+      if (inputs.ENABLE_NETWORK_AIR_GAP) {
+        if (inputs.SYNOPSYS_BRIDGE_PATH.length !== 0) {
+          executable = await this.setBridgeExecutablePath(
+            osName,
+            inputs.SYNOPSYS_BRIDGE_PATH
+          );
+          if (!taskLib.exist(executable)) {
+            throw new Error(
+              "synopsys_bridge_path ".concat(
+                inputs.SYNOPSYS_BRIDGE_PATH,
+                " does not exists"
+              )
+            );
+          }
+        } else {
+          executable = await this.setBridgeExecutablePath(
```

Comment:
can we validate default bridge path and log the error if it doesn't exist
and if executable not found, throw error relatable to that